packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v2.3.0-deploy-contract-from-json-private.test.ts

```diff
@@ -0,0 +1,278 @@
+  const pluginRegistry3 = new PluginRegistry();
+
+  const pluginFactoryLedgerConnector = new PluginFactoryLedgerConnector({
+    pluginImportType: PluginImportType.Local,
+  });
+
+  const keychainInstanceId1 = "keychain_instance1_" + uuidV4();
+  const keychainId1 = "keychain1_" + uuidV4();
+  const keychain1 = new PluginKeychainMemory({
+    instanceId: keychainInstanceId1,
+    keychainId: keychainId1,
+    logLevel,
+  });
+  t.ok(keychain1, "keychain1 truthy OK");
+  keychain1.set(
+    HelloWorldContractJson.contractName,
+    JSON.stringify(HelloWorldContractJson),
+  );
+  pluginRegistry1.add(keychain1);
+
+  const keychainInstanceId2 = "keychain_instance2_" + uuidV4();
+  const keychainId2 = "keychain2_" + uuidV4();
+  const keychain2 = new PluginKeychainMemory({
+    instanceId: keychainInstanceId2,
+    keychainId: keychainId2,
+    logLevel,
+  });
+  t.ok(keychain2, "keychain2 truthy OK");
+  keychain2.set(
+    HelloWorldContractJson.contractName,
+    JSON.stringify(HelloWorldContractJson),
+  );
+  pluginRegistry2.add(keychain2);
+
+  const keychainInstanceId3 = "keychain_instance3_" + uuidV4();
+  const keychainId3 = "keychain3_" + uuidV4();
+  const keychain3 = new PluginKeychainMemory({
+    instanceId: keychainInstanceId3,
+    keychainId: keychainId3,
+    logLevel,
+  });
+  t.ok(keychain3, "keychain3 truthy OK");
+  keychain3.set(
+    HelloWorldContractJson.contractName,
+    JSON.stringify(HelloWorldContractJson),
+  );
+  pluginRegistry3.add(keychain3);
+
+  const connectorInstanceId1 = "quorum1_" + uuidV4();
+  const connectorInstanceId2 = "quorum2_" + uuidV4();
+  const connectorInstanceId3 = "quorum3_" + uuidV4();
+
+  const connector1 = await pluginFactoryLedgerConnector.create({
+    instanceId: connectorInstanceId1,
+    pluginRegistry: pluginRegistry1,
+    rpcApiHttpHost: rpcApiHttpHostMember1,
+    privateUrl: keys.quorum.member1.privateUrl,
+    logLevel,
+  });
+  t.ok(connector1, "connector1 truthy OK");
+  test.onFinish(() => connector1.shutdown());
+  pluginRegistry1.add(connector1);
+
+  const connector2 = await pluginFactoryLedgerConnector.create({
+    instanceId: connectorInstanceId2,
+    pluginRegistry: pluginRegistry2,
+    rpcApiHttpHost: rpcApiHttpHostMember2,
+    privateUrl: keys.quorum.member3.privateUrl,
+    logLevel,
+  });
+  t.ok(connector2, "connector2 truthy OK");
+  test.onFinish(() => connector2.shutdown());
+  pluginRegistry2.add(connector2);
+
+  const connector3 = await pluginFactoryLedgerConnector.create({
+    instanceId: connectorInstanceId3,
+    pluginRegistry: pluginRegistry3,
+    rpcApiHttpHost: rpcApiHttpHostMember3,
+    privateUrl: keys.quorum.member3.privateUrl,
+    logLevel,
+  });
+  t.ok(connector3, "connector3 truthy OK");
+  test.onFinish(() => connector3.shutdown());
+  pluginRegistry3.add(connector3);
+
+  await connector1.onPluginInit();
+  await connector2.onPluginInit();
+  await connector3.onPluginInit();
+
+  const accountKeyPath = path.resolve(
```

Comment:
@travis-payne Now I see where that folder is used. Can the credentials get pulled from the container instead? (See `Containers.pullFile(...)`)

Reply:
Fixed!